src/templates/landing-page.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { graphql } from "gatsby";
+import { GridContainer } from "@trussworks/react-uswds";
+import { Graphic, Header, Hero, Section, Tagline } from "../components";
+import "@trussworks/react-uswds/lib/uswds.css";
```

Comment:
Are these imported in `index.js` as well?

Reply:
I believe we won't use index.js going forward and will use landing-page instead, so I imported here. Also, if we were to have multiple pages, we might need to import those multiple times, since the file structure is different than with create-react-app and we don't import all the other pages into index.js eventually. But we should look into that!